client/src/redux/home/service.js

```diff
@@ -28,6 +28,24 @@ const getList = async params => {
 
   return data;
 };
+
+const getPaginatedList = async pageNumber => {
+  const response = await fetch(`${BASE_URL}/home/${pageNumber}`, {
```

Comment:
page number should be sent as a query parameter with filter options through home GET router
ex) http://localhost:3000/home?location=WE+Downtown+Vancouver&minPrice=0&maxPrice=0&bedsMin=1&home_type=House&page=3

`getPaginatedList` doesn't cooperate with filter options
please embed pageNumber into getList function's `params` argument